docs/api_call.md

```diff
@@ -1,1352 +1,1108 @@
 
-  - Example 1: 
+1. **message**: Message on which detection logic needs to run. It is an unstructured text from which entity needs to be extracted. For example, *"I want to order pizza"*.
 
-    - ```python
-      message='i want to order chinese from  mainland china and pizza from domminos'
-      entity_name='restaurant'
-      structured_value=None
-      fallback_value=None
-      bot_message=None
-      ```
-
-    - *Python:* 
-      ```python
-      from ner_v1.chatbot.entity_detection import get_text
-      output = get_text(message=message, entity_name=entity_name, structured_value=structured_value, fallback_value=fallback_value, bot_message=bot_message)
-      print output
-      ```
-      The above can also be done from within the Docker container's shell. Setup is in docker.md file.
-
-    - *CURL command:*
-      ```shell
-      URL='localhost'
-      PORT=8081
-      ```
-
-      ```shell
-      curl -i 'http://'$URL':'$PORT'/v1/text/?message=i%20want%20to%20order%20chinese%20from%20%20mainland%20china%20and%20pizza%20from%20domminos&entity_name=restaurant&structured_value=&fallback_value=None&bot_message='
-      ```
-
-    - *CURL Output:*
-
-      ```json
-      {
-        "data": [
-          {
-            "detection": "message",
-            "original_text": "mainland china",
-            "entity_value": {
-              "value": "Mainland China"
-            }
-          },
-          {
-            "detection": "message",
-            "original_text": "dominos",
-            "entity_value": {
-              "value": "Domino's Pizza"
-            }
-          }
-        ]
-      }
-      ```
-
-  - Example 2: 
-
-    - ```python
-      message = 'i wanted to watch movie'
-      entity_name = 'movie'
-      structured_value = 'inferno'
-      fallback_value = None
-      bot_message = None
-      ```
-
-    - *Python:* 
-
-      ```python
-      from ner_v1.chatbot.entity_detection import get_text
-      output = get_text(message=message, entity_name=entity_name, structured_value=structured_value, fallback_value=fallback_value, bot_message=bot_message)
-      print output
-      ```
-
-    - *CURL command:*
-
-      ```shell
-      URL='localhost'
-      PORT=8081
-      ```
-
-      ```shell
-      curl -i 'http://'$URL':'$PORT'/v1/text/?message=i%20wanted%20to%20watch%20movie&entity_name=movie&structured_value=inferno&fallback_value=None&bot_message='
-      ```
-
-    - *CURL Output*:
-
-    - ​
-
-      ```json
-      {
-        "data": [
-          {
-            "detection": "structure_value_verified",
-            "original_text": "inferno",
-            "entity_value": {
-              "value": "Inferno"
-            }
-          }
-        ]
-      }
-      ```
-
-  - ​
-
-
-### phone_number
+2. **entity_name**: Name of the entity. This parameter is important when the detection logic relies on dictionary. It is used to run particular detection logic by looking at its dictionary.
```

Comment:
This needs simplification and rewriting. "This parameter primarily defines the key of the output dictionary. (Give example)
It has no other significance except for text entity, etc. "